.chronus/changes/Issue3601-2024-6-18-17-42-35.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: fix
+packages:
+  - "@typespec/openapi3"
+---
+
+Add '$ref' to the model. Refer [3601](https://github.com/microsoft/typespec/pull/3894) for further details.
```

Comment:
```suggestion
Add support for `@useRef` on responses
```

Do not link the pr it will get auto linked in the changelog at release, also this should be descriptive enough for people to not have to go to the PR and get a good idea of what changed